docs/tutorials/embedded/wine_detector.md

```diff
@@ -10,30 +10,29 @@
 
 If there is another similar tutorial that's more appropriate for another audience, direct the reader there with a linked reference. -->
 
-### What's In This Tutorial?
-In this demo we will walk through the steps to setup a real-time object classifier, powered by MxNet running entirely on a Raspberry Pi 3. We will then work through connecting this local Raspberry Pi model to the AWS cloud via the IoT service, letting you easily and reliablely get realtime updates from the device.
+## What's In This Tutorial?
+In this demo we will walk through the steps to set up a real-time object classifier, powered by MxNet running entirely on a Raspberry Pi 3. We will then work through connecting this local Raspberry Pi model to the AWS cloud via the IoT service, letting you easily and reliably get real-time updates from the device.
```

Comment:
+This tutorial shows how to:

1. Use MXNet to set up a real-time object classifier on a Raspberry Pi 3 device. 

2. Connect the local Raspberry Pi model to the AWS Cloud with AWS IoT to get real-time updates from the device.